graphene_django/tests/test_fields.py

```diff
@@ -0,0 +1,42 @@
+import pytest
+
+from graphene import ObjectType, Schema
+from graphene.relay import Node
+from graphene_django import DjangoConnectionField, DjangoObjectType
+from graphene_django.tests.models import Article, Pet, Reporter
+
+pytestmark = pytest.mark.django_db
+
+
+def test_required_connection_field():
```

Comment:
What is the difference between this test and test_required_filter_field ?

Reply:
@zbyte64 this test uses a `DjangoConnectionField` while the other test uses a `DjangoFilterConnectionField`